heroku-18-cnb-build/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM heroku/heroku:18-build
+
+ARG pack_uid=1000
+ARG pack_gid=1000
```

Comment:
Yeah, I think I put these in so it was easier to change in one place. 

I think it's probably good to drop them, though. It would reduce any chance of confusion. These really are constants, and not something that should be passed in.